pyproject.toml

```diff
@@ -13,7 +13,7 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-pytest = [
```

Comment:
since you changed the name on this, you need to update the github workflow to install testing